lib/logstash/filters/translate.rb

```diff
@@ -62,11 +65,20 @@ class LogStash::Filters::Translate < LogStash::Filters::Base
   #     merci: gracias
   #     old version: new version
   #     
-  # NOTE: it is an error to specify both `dictionary` and `dictionary_path`
+  # NOTE: it is an error to specify both `dictionary` and `dictionary_path` or `dictionary_url`
   config :dictionary_path, :validate => :path
 
-  # When using a dictionary file, this setting will indicate how frequently
-  # (in seconds) logstash will check the YAML file for updates.
+  # The full URI path of a Web service who generates an yml format response. 
+  # The response generated needs to be equals than needed for @dictionary_path
+  #
+  # NOTE: it is an error to specify both `dictionary` and `dictionary_path` or `dictionary_url`
+  config :dictionary_url, :validate => :string
+
+  # Filename (without extension) where .yml will be stored from a Web service.
+  config :file_to_download, :validate => :string, :default => "dictionary"
```

Comment:
suppose you have two translate "instances", sometimes the webservice endpoint will be the same but you need to save the file separately, here you can define how the file will be named without the risk to one overwrites another.